server/events/init.js

```diff
@@ -1,25 +1,37 @@
+const url = require('url')
 const logger = require('winston')
+const { setupCookie, handleCustomerOnload, addFeatureToExistingUser } = require('../util/init/init-utility')
 
 const initEvent = async (ctx) => {
+  const isMock = false
   const { body } = ctx.request
-  const { hashedUserId } = body
-  const { customerCode } = body
-  const responseString = 'function(a,b,c,d,e){a.ic=function(b){a.i=b},d=b.createElement(c),e=b.getElementsByTagName(c)[0],d.async=!0,d.src="http://localhost:3000/analytics.js",e.parentNode.insertBefore(d,e)}(window,document,"script"),ic("IC9-55938-5"' + ', "' + customerCode + '");'
+  const customerCode = body.customerCode
+  const { hashedUserId } = body.userIdentity || {}
+  const hostname = url.parse(ctx.request.origin).hostname
+  const cookie = await setupCookie((body.userIdentity || {}).deviceCode)
+  // const responseString = 'function(a,b,c,d,e){a.customerCode=function(b){a.i=b},d=b.createElement(c),e=b.getElementsByTagName(c)[0],d.async=!0,d.src="http://localhost:3000/analytics.js",e.parentNode.insertBefore(d,e)}(window,document,"script"),customerCode("' + customerCode + '", "' + hashedUserId + '");'
+  const responseString = 'console.log(\'Hello I\\\'m interact\')'
+  const responseObjMock = {'featureList': [ {'name': 'Card-1', 'version': 'A'}, {'name': 'Card-2', 'version': 'B'} ], 'deviceCode': 'test', 'initCode': responseString}
```

Comment:
I think we shouldn't include the mock object inside this method (also the real `initCode`, the one you called it `responseString`, too)

To get `initCode`, may be we need `function getInitCode(params)`